gc-policy.yaml

```diff
@@ -1,15 +1,31 @@
+# Production
+testing:
+    cloud-uploads: 2y
+    containers: 2m
+stable:
+    cloud-uploads: 2y
+    containers: 2m
+next:
+    cloud-uploads: 2y
+    containers: 2m
+# Development
 testing-devel:
     cloud-uploads: 2y
     containers: 2w
 next-devel:
     cloud-uploads: 2y
     containers: 2w
-branched:
+# Mechanical
+rawhide:
     cloud-uploads: 1y
     containers: 2w
-rawhide:
+    images: 3y
+    build: 4y
```

Comment:
for rawhide I would probably do images 2y build 3y - so reduce a year off of both of these